new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWITValidation.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes;
+
+import oracle.weblogic.kubernetes.actions.TestActions;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static oracle.weblogic.kubernetes.actions.TestActions.withWITParams;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@DisplayName("Simple validation of basic WIT functions")
+class ItWITValidation implements LoggedTest {
+  @Test
+  @DisplayName("Create a MII image")
+  public void testCreatingMIIImage() {
+
+    // install WIT using the default version and location
+    boolean downloadWIT = TestActions.installWIT(
+        null /* version */, 
```

Comment:
use //

Reply:
This comment does not exist any more in the latest.